quick/mainqt5.cpp

```diff
@@ -9,6 +9,8 @@
 
 int main(int argc, char **argv)
 {
+    setenv("MOZ_LAYERS_PREFER_OFFSCREEN", "1", 1);
```

Comment:
did you mean MOZ_USE_EXTERNAL_WINDOW here? MOZ_LAYERS_PREFER_OFFSCREEN already enabled in embedlite